src/main/java/sparksoniq/spark/ml/GetEstimatorFunctionIterator.java

```diff
@@ -112,7 +112,7 @@ public Item next() {
                 List<SequenceType> paramTypes = Collections.unmodifiableList(
                     Arrays.asList(
                         new SequenceType(
-                                new ItemType(ItemTypes.Item), // TODO: revert back to ObjectItem
```

Comment:
This causes iterator to be wrapped again in TypePromotionIterator. Since this TPI is only RDD enabled(no DF implementation yet), the results are writtent as an RDD to the DynamicContext. This creates the need of reading out RDD or local variables from DC, converting them to a DataFrame of primitives, so that they can be passed into SparkML to be processed.